docs/quickstarts/javascript.mdx

```diff
@@ -36,7 +36,7 @@ To add the [ClerkJS SDK](/docs/references/javascript/overview) to your JavaScrip
 
 Select your preferred method below to get started.
 
-<Tabs items={["NPM module", "<script>"]}>
+<Tabs type="npm-script" items={["NPM module", "<script>"]}>
```

Comment:
This change is fine, but this page renders all the steps in the ToC, even though half of them are scoped to one tab and half are scoped to the other. Everything below the red line in this screenshot is hidden if "NPM module" is selected, but still is in the ToC:

<img width="227" alt="Screenshot 2024-03-05 at 12 23 55 PM" src="https://github.com/clerk/clerk-docs/assets/5156915/77b46d3d-cac6-46a3-9a6f-38a1db745795">

Reply:
ugh how annoying - so @jescalan and I did talk about splitting the JS quickstart into two separate tutorial pages in the future, but the `<Tabs />` is our solution for now - possibly after the IA, we can think about this again. what do you think?